ui/analyse/src/ceval/sunsetterProtocol.js

```diff
@@ -87,14 +54,12 @@ module.exports = function(worker, opts) {
     start: function(w) {
       stopped = null;
       work = w;
-      worker.send('reset crazyhouse');
       worker.send('setboard ' + work.initialFen);
-      worker.send('easy');
-      worker.send('force');
+      worker.send('exit');
```

Comment:
oh, til :+1: . i am pretty sure i made this mistake when i tried to adapt sunsetter for server side analysis